storage/remote/queue_manager.go

```diff
@@ -1018,9 +1065,10 @@ type shards struct {
 	qm     *QueueManager
 	queues []*queue
 	// So we can accurately track how many of each are lost during shard shutdowns.
-	enqueuedSamples    atomic.Int64
-	enqueuedExemplars  atomic.Int64
-	enqueuedHistograms atomic.Int64
+	enqueuedSamples         atomic.Int64
+	enqueuedExemplars       atomic.Int64
+	enqueuedHistograms      atomic.Int64
+	enqueuedFloatHistograms atomic.Int64
```

Comment:
Let's track all the numbers in the existing histogram metrics itself (here and elsewhere). 'float histogram' is just a different in-memory representation of native histogram while at the end they are just histograms.